org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/SynapseLanguageService.java

```diff
@@ -304,6 +304,12 @@ public void updateConnectors() {
         SchemaGenerate.generate(connectorHolder, connectorPath);
     }
 
+    public void updateInboundConnectors() {
+
+        inboundConnectorHolder.getCustomInboundConnectors(new File(Path.of(projectUri, Constant.SRC, Constant.MAIN, Constant.WSO2MI,
+                Constant.RESOURCES, Constant.INBOUND_CONNECTORS_DIR).toString()), projectServerVersion);
```

Comment:
Do we need to pass the inbound connector directory? It can be made as a constant in the InboundConnectorHolder, right?

Reply:
Fixed in f2945517fa222e1e1375ddfaa5cb92a45d5605b5